TopAnalysis/data/grouping/fcncTrilepton.yaml

```diff
@@ -63,7 +63,7 @@ processes:
     DYJets:
         title: "DYJets"
         longTitle: "Z/#gamma^{*}+Jets#rightarrow l^{+}l^{-}"
-        weight: "(LHEWeight_originalXWGTUP == 0 ? genWeight : genWeight/abs(LHEWeight_originalXWGTUP))*puWeight*BtagWeight*LeptonSF"
+        weight: "LHEScaleWeight[4]*genWeight/abs(genWeight)*puWeight*BtagWeight*LeptonSF"
```

Comment:
LHEScaleWeight[4]/abs(LHEWeight_originalXWGTUP)*puWeight*BtagWeight*LeptonSF

Reply:
LHEScaleWeight = LHEEvent.weight.wgt/LHEWeight_originalXWGTUP
So if we divide LHEScaleWeight by abs(LHEWeight_originalXWGTUP), then w_nominal (original weight) will be divided twice...